src/main/java/spleetwaise/transaction/logic/parser/ParserUtil.java

```diff
@@ -79,6 +83,31 @@ public static Date parseDate(String date) throws ParseException {
         return new Date(date);
     }
 
+    /**
+     * Parses a {@code String catStr} into a {@code Category} that represents the category. Leading and trailing
+     * whitespaces will be trimmed and capitalized
+     */
+    public static Category parseCategory(String catStr) throws ParseException {
+        requireNonNull(catStr);
+        String trimmedCategory = catStr.trim().toUpperCase();
+        if (!Category.isValidCatName(trimmedCategory)) {
+            throw new ParseException(Category.MESSAGE_CONSTRAINTS);
+        }
+        return new Category(trimmedCategory);
+    }
+
+    /**
+     * Parses {@code Collection<String> Category} into a {@code Set<Category>}.
+     */
+    public static HashSet<Category> parseCategories(Collection<String> categoryStrs) throws ParseException {
+        requireNonNull(categoryStrs);
+        final HashSet<Category> categories = new HashSet<>();
```

Comment:
```suggestion
        Set<Category> categories = new HashSet<>();
```
Standardise to the same as previously mentioned.

Reply:
👍 Made the change